WormsApp/MapperWrapper.h

```diff
@@ -0,0 +1,18 @@
+#ifndef MAPPERWRAPPER_H
+#define MAPPERWRAPPER_H
+
+#include <unordered_map>
+#include <utility>
+#include<thread>
+
+template <typename KeyType, typename valueType> class MapperWrapper
+{
+    public:
+        void add(KeyType key, valueType&& value)
+        {
+        mapper.emplace(key, std::forward<std::thread>(value));
+        }
+
+        std::unordered_map<KeyType, valueType> mapper;
```

Comment:
This is a public variable of a class. Usually you want to hide your internal variables so that no one else from outside can modify (and possibly brake) them. If you want to access such variable you create a getter function that returns a constant reference to such variable or creates a copy and returns it.

Returning const reference (another const after the method name means we don't modify our class):
```
const std::unordered_map<KeyType, valueType>& getMapper() const {
  return mapper;
}
```

Copy the map (not recommended - it's time consuming and may mess up our threads):
```
std::unordered_map<KeyType, valueType> getMapper() const {
  return mapper;
}
```

Reply:
Getter is added.